index.bs

```diff
@@ -1866,6 +1866,13 @@ usage.
       not be found.
     </td>
   </tr>
+  <tr>
+    <td>{{NotReadableError}}</td>
```

Comment:
Might be worth mentioning that this is considered a "permanent" loss or that the data is "irrecoverable"?

(Sorry for late review)

Reply:
Thanks, I'll update this in a follow-up PR.